src/FMBot.Bot/SlashCommands/DiscogsSlashCommands.cs

```diff
@@ -61,16 +62,18 @@ public async Task DiscogsAsync()
     [UsernameSetRequired]
     public async Task AlbumAsync(
         [Summary("Search", "Search query to filter on")] string search = null,
-        [Summary("User", "The user to show (defaults to self)")] string user = null)
+        [Summary("User", "The user to show (defaults to self)")] string user = null,
+        [Summary("Formats", "Formats to include")] params string[] formats)
```

Comment:
This can be an enum, then you get a nice dropdown. Some other slash commands already have this

Reply:
so basically this? how could one select multiple in one go?

```diff
- [Summary("Formats", "Formats to include")] params string[] formats)
+ [Summary("Formats", "Formats to include")] DiscogsFormat formats)
```

Reply:
yeah, you need to have a default though (otherwise it becomes a required parameter), so like

```[Summary("Formats", "Formats to include")] DiscogsFormat? format = null)```

Also Discord doesn't support selecting multiple at the moment. But having it is as a dropdown is more intuitive imo, so worth the tradeoff